main/integration-tests/support/setup.js

```diff
@@ -83,6 +86,12 @@ class Setup {
     return defaultProject.indexId;
   }
 
+  async getDefaultAwsAccountId() {
+    const adminSession = await this.defaultAdminSession();
+    const defaultIndex = await adminSession.resources.indexes.index(this.defaultIndexId).get();
+    return defaultIndex.awsAccountId;
+  }
+
```

Comment:
This is just a general comment regarding adding more initialization logic to setup. We need to keep a few things in mind:

- setup is called per test suite (test file)
- if there is anything wrong with the initialization (say there is a bug in the index api), then none of the tests will run

So it is a matter of figuring out the balance.  For now, I think these new additions are fine, but we just need to keep the above two points in  mind.

Reply:
Since these are the default project/index/awsAccount they would be found unless the provided projectId was invalid. I agree, there is room for making it more efficient.